### PR TITLE
Fix some typos

### PR DESCRIPTION
**Pull request by paulo-ferraz-oliveira**

### CODE REVIEW

File: content/en/docs/commands.md [hunk 1]
```diff
@@ -5,15 +5,15 @@ description: >
     Usage of each rebar3 command.
 ---
 
-Each command represents a task which runs one or more providers to fulfill the task.
+Each command represents a task which runs one or more providers to fulfil the task.
```
Comment (ferd) [inline content/en/docs/commands.md:8]:
```suggestion
Each command represents a task which runs one or more providers to fulfill the task.
```
I'd probably keep the US variant here.

**Review by ferd — APPROVED** [review]
Thanks for this. There's one part where I'd favor/keep the US spelling but the rest is all good.

File: content/en/docs/commands.md (hunk 1 above)
Comment (paulo-ferraz-oliveira) [reply content/en/docs/commands.md:8]: Oh, I actually thought that WAS the US variant :) sorry about that. TIL

Comment (paulo-ferraz-oliveira) [reply content/en/docs/commands.md]: I knew I wanted to keep US for the use of "z" instead of "s", everywhere... 😄

**Review by ferd — APPROVED** [review]

**Pull request merged**